src/Components/Projects/Projects.js

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import "./Projects.css";
+import PokeBattle from "../../Images/pokemonBattle.png";
+import AnimeApp from "../../Images/AnimeApp.png";
+import WhatToEat from "../../Images/WhatToEat.png";
+import Calculator from "../../Images/Calculator.png";
+import Plane from "../../Images/plane.webp";
+import PKMNCard from "../../Images/pokemoncard.png";
+
+export default function Projects() {
+    return (
+        <section id="projects" className="projectsSection">
+            <h2 className="projectsTitle">Projects</h2>
+            <hr className="horizontalRule"></hr>
+            <ul className="projectsContainer">
+                <li className="project">
+                    <a
+                        href="https://chipper-otter-f36709.netlify.app/"
+                        className=""
+                    >
+                        <img src={PKMNCard} alt="" className="icon" />
+                    </a>
+                    <a href="https://chipper-otter-f36709.netlify.app/">
+                        <h3>Pokemon Card Database</h3>
+                    </a>
+
+                    <p>
+                        Uses a Django REST API to store pokemon cards. Users can
```

Comment:
You may want to change this text a bit to mention that the data is stored in a sqlite db and accessed/modified via a Django Rest API